packages/terra-date-picker/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Unreleased
 
 4.40.0 - (June 16, 2020)
 ------------------
+### Added
```

Comment:
whoops, needs move to under unreleased section

Reply:
moved to unreleased : [8a57cfd5b](https://github.com/cerner/terra-framework/pull/1134/commits/8ec0355f75cba2e79d27fbd326b343bec97afba4#diff-0e615a87d99527850fd6ccd8a57cfd5b)